test/Makefile.am

```diff
@@ -32,6 +33,7 @@ test_t8_test_half_neighbors_SOURCES = test/t8_test_half_neighbors.cxx
 test_t8_test_element_count_leafs_SOURCES = test/t8_test_element_count_leafs.cxx
 test_t8_test_search_SOURCES = test/t8_test_search.cxx
 test_t8_test_find_parent_SOURCES = test/t8_test_find_parent.cpp
+test_t8_test_successor_SOURCES = test/t8_test_successor.cpp
```

Comment:
C++ files should be named *.cxx not *.cpp.
I missed this one for find_parent as well.